azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -677,4 +680,30 @@ protected void handleMultiformPost(final HttpServletRequest req,
       final HttpServletResponse resp, final Map<String, Object> multipart, final Session session)
       throws ServletException, IOException {
   }
+
+  /**
+   * Method to check permission to access image management APIs.
+   *
+   * @param imageTypeName
+   * @param user
+   * @param type
+   * @return boolean
+   */
+  protected boolean hasImageManagementPermission(final String imageTypeName, final User user,
+      final Permission.Type type) {
+    final UserManager userManager = getApplication().getUserManager();
+    for (final String roleName : user.getRoles()) {
+      final Role role = userManager.getRole(roleName);
+      /**
+       * Azkaban ADMIN role must have full permission to access image management APIs. Hence, no
+       * further permission check is required.
+       */
+      if (role.getPermission().isPermissionSet(Permission.Type.ADMIN)) {
```

Comment:
Can't member access these APIs? Is it only allowed to Admin?

Reply:
Permission.Type.ADMIN referred here is Azkaban Admin. On the other hand, the ownership defined as part of image type has owners with roles such as ADMIN and MEMBER. So, initially only Azkaban admin (Azkaban dev or service account admin defined in azkaban-users.xml) can create a new image type with owners. Once the owners are added for  an image type they can access the image management APIs. Owner with ADMIN role can access all the image management APIs including add/remove image type owners (This two api to be added in future). Owner with member role can access all the APIs except add/remove image type owners.